### PR TITLE
[MMCA-4446] - Remove unused code and ToDos through out the services

### PR DESCRIPTION
**Pull request by tyagikumarvineet**
Description - Prepare repo for live services handover

Below has been done as part of this PR

- remove unused code including unused parameters
- add scalastyle configs for both code and test
- remove descriptive comments
- remove comments
- remove TODO and enable tests wherever applicable
- refactoring to make the code readable
- refactored long lines

- remove ignored tests for CashTransactionsRequestPageFormProviderSpec as those seems have become obsolete in terms of requirement but can be taken care as part of separate ticket 
        
- remove dangling strings
- remove magic numbers

- cyclomatic complexity has been increased to 11 as few methods in the code are over complexity 10 and it needs quite a bit of time to refactor and testing. 
- update sbt-auto-build version to 3.20.0
- update to build.sbt
- unused application.conf entry has also been removed

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -128,7 +128,6 @@ urls {
   signOut = "http://localhost:9553/bas-gateway/sign-out-without-state"
   customsFinancialsHomepage = "http://localhost:9876/customs/payment-records"
   govUkHome = "https://www.gov.uk"
```
Comment (harshare) [inline conf/application.conf:130]: @tyagikumarvineet , noticed few unused items in the config. Can you please remove these items?
govUkHome
application.cash-account.updateTime.intervalMilliseconds 
application.cash-account.updateTime.timeoutMilliseconds

File: app/controllers/UnauthorisedController.scala [hunk 2]
```diff
@@ -23,12 +23,12 @@ import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import views.html.unauthorised
 
-class UnauthorisedController @Inject()(
-                                        controllerComponents: MessagesControllerComponents,
-                                        unauthorisedView: unauthorised
-                                      )(implicit val appConfig: AppConfig) extends FrontendController(controllerComponents) with I18nSupport {
+class UnauthorisedController @Inject()(controllerComponents: MessagesControllerComponents,
+                                        unauthorisedView: unauthorised)
```
Comment (harshare) [inline app/controllers/UnauthorisedController.scala:27]: @tyagikumarvineet , there is an extra space before the parameter on line 27. Can you please reformat this file?

Comment (tyagikumarvineet) [reply app/controllers/UnauthorisedController.scala]: Reformatted

**Review by harshare — APPROVED** [review]
LGTM

**Pull request merged**